spec/support/features/localization_helper.rb

```diff
@@ -4,6 +4,14 @@ def invalid_phone_message
       t('errors.messages.improbable_phone')
     end
 
+    def unsupported_phone_message
```

Comment:
what is this used for?

Reply:
The specs...that didn't get included in that push

Reply:
Also, yeah adding some controller specs as well.